flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -1085,27 +1081,27 @@ public void testNotifyCheckpointOnClosedOperator() throws Throwable {
 
         harness.streamTask.notifyCheckpointCompleteAsync(1);
         harness.streamTask.runMailboxStep();
-        assertEquals(1, ClosingOperator.notified.get());
-        assertFalse(ClosingOperator.closed.get());
+        assertThat(ClosingOperator.notified.get()).isOne();
```

Comment:
IIRC, AssertJ does have specific assertions for `AtomicXXX`, please avoid using `get`.
It is also necessary to check the similar problems of the whole test class.

Reply:
Thank you~. I updated it.